### PR TITLE
Clean up css in dblog

### PR DESCRIPTION
**Pull request by adamtothemoon**
pr for: https://github.com/backdrop/backdrop-issues/issues/207

### CODE REVIEW

File: core/modules/dblog/dblog.css [hunk 1]
```diff
@@ -15,50 +15,50 @@
    */
   width: 15em;
 }
-#dblog-filter-form .form-type-select select {
+.dblog-filter-form .form-type-select select {
   width: 100%;
 }
-#dblog-filter-form .form-actions {
+
```
Comment (jenlampton) [inline core/modules/dblog/dblog.css:21]: Looks like you are missing

```
+.dblog-filter-form .form-actions {
```

**Comment by jenlampton** [comment]
This looks very close!  If you can fix up that one omission it should be good to go.

**Comment by jenlampton** [comment]
Closing PR since it needs work.  Please re-open when ready for a new review.

**Comment by jenlampton** [comment]
Reopening since PRs closed by me end up locked :(

**Comment by quicksketch** [comment]
Filed a replacement PR for this at https://github.com/backdrop/backdrop/pull/331.

**Pull request closed without merging**